users/ninjonas/process_records.c

```diff
@@ -81,7 +77,16 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
     // Opens Visual Studio Code on current directory
     case M_CODE:
       if (record->event.pressed) {
-        SEND_STRING("code ." SS_TAP(X_ENTER));
+        SEND_STRING("code .\n");
+      }
+      break;
+
+    // Opens Terminal
+    case M_TERM:
+      if (record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_LGUI) SS_TAP(X_SPACE) SS_UP(X_LGUI));
+        _delay_ms(250);
```

Comment:
This would be more "future-proof". `_delay_ms()` only works on AVR-based keyboards, while `wait_ms()` is platform-agnostic (works on AVR or ARM architecture).

```suggestion
        wait_ms(250);
```

May not matter now, but depending on future keyboards you buy, it could matter later.

Please advise or commit if you want to change to this, or just let me know that you want to go ahead and have this merged as-is.

Reply:
thank you @noroadsleft! just pushed the requested change.